---
layout: default
title: 7. Preparing high titer lysates for DNA extraction
parent: Lab Protocols
nav_order: 7
published: true
---

# Preparing high titer lysates for DNA extraction
{: .no_toc }

<details open markdown="block">
  <summary>
    Table of contents
  </summary>
  {: .text-delta }
1. TOC
{:toc}
</details>

## Objective
{: .no_toc }
To acquire a high concentration of viral particles to have enough material for whole genome sequencing of the phages.

## Rationale
{: .no_toc }
Each phage is unique and encodes its own set of genes that is variable between phage phamilies and phages that infect different bacterial hosts. In addition to plaque assays, which tells us information about the phenotypes of these phages (such as plaque size and clear-ness), we also rely on whole genome sequencing to inform us about the genotypes of our phages, such as which genes these phages encode and what makes one phage different from another on a nucleotide level. In order to be able to sequence the genomes of the phages, we first need to ensure that we have enough phage particles by creating a high titer phage lysate. These high titer (HT) lysates can be collected by flooding plates with as many plaques as possible. These plates are called confluent lysis plates, or “webbed” plates, because they have densely packed confluent plaques with only wisps, or a “web” of bacteria, left between them. Depending on the phage, each plaque on a plate may contain between 103 and 107 phage particles. These phage are collected to generate a concentrated phage solution called a lysate. Lysates are collected by “flooding” a webbed plate. Flooding is accomplished by adding SM buffer to cover the surface of the plate and allowing the phage to diffuse into the buffer. The buffer is then harvested and filtered to remove agar particles and bacteria. We then will do a quick spot titer to estimate the PFU of the HT stock to determine if there are enough phages to perform whole genome sequencing.

---

## Making _webbed_ confluent lysis plates

### Before you start:
{: .no_toc }
- You need a growing culture of your bacterial host
- You need to have pre-melted molten top agar in your [bead bath]
- Add ions to the top agar to facilitate phage adsorption. (10 mM CaCL2 or MgSO4)
- You need to have an estimate of the titer of your phage of interest

### Materials making “webbed” confluent lysis plates
{: .no_toc }
- X number of LB agar plates (as many plates as samples to be plaqued + 1 no phage control)
- X number of test tubes (as many tubes as samples to be plaqued + 1 for no phage control)
- LB top agar
- Serological pipettes
- Microcentrifuge tubes
- SM buffer
- 1 mL syringes
- 0.22 uM filters for syringes

--

### Procedure for making “webbed” confluent lysis plates
{: .no_toc }

#### Day 1
{: .no_toc }

1. Calculate the amount of phage needed to create a “webbed” confluent lysis plate
1. Using titers from the plaque assay, observe the appropriate dilution that gave you webbed-looking plates. Each phage is different, with different plaque sizes, therefore you need to customize your concentrations for each phage you test.
1. Alternatively, estimate how much phage you would need to add to give you 10,000 plaques per plate.
1. Multiple different dilutions can be tested to see what gives you the optimum plaque density
1. Label LB agar plates with today's date, your name, your phage name, and dilutions to be plated, including a no phage control plate.
1. Set out as many test tubes as samples you will be plating. Label the culture tubes with the phage name and the phage dilution, including a no phage control tube.
1. Add 250 uL of the appropriate bacterial host to each labeled tube.
1. Add the appropriate volume and dilution of phage to each tube, as well as 10 uL of SM buffer with no phage as a control. Gently vortex the tubes and allow to sit on the benchtop undisturbed for 10 minutes to allow phage adsorption
1. Using a sterile serological pipette, gently add 3 mL molten agar supplemented with CaCl2 to a single inoculated host, and immediately suck up the top agar/phage/bacteria mixture and transfer it to the appropriately labeled plates. Be sure to avoid air bubbles, as they can look like plaques on plates.
1. Quickly tilt the plate in multiple directions until the top agar mixture evenly coats the agar plate. 
1. Repeat this process for all of your samples, including the no phage control.
1. Let the plates sit undisturbed for 20 minutes on the benchtop to allow the top agar to fully solidify.
1. Without inverting, incubate the plates overnight in the appropriate incubator.

#### Day 2
{: .no_toc }

1. Retrieve the plates from the incubator and identify which plate(s) show the best evidence of confluent lysis. Write down the conditions in your lab notebook (dilution/volume)
2. Flood the lysis plates with 8 mL SM buffer, and let the plates rock at room temperature for 3 hours *or overnight at 4 C??* → also probably won’t have enough time to do the overlay, collect, filter, and spot all in one day so I’m leaning towards rocking O/N

#### Day 3 (or afternoon of day2)
{: .no_toc }

1. Label two microcentrifuge tubes for each phage.
1. Retrieve the overlain plates gently.
1. Remove the lid from the plate and place it on the bench. Tilt the plate slightly by placing one edge of the plate on the lid, allowing the lysate to pool to one side.
1. Collect one mL of lysate into the appropriately labeled microcentrifuge tube.
1. Centrifuge the cells to remove and bacteria for 3 minutes at 5,000xg.
1. Remove the plunger from a 1 mL syringe and attach a filter to the tip.
1. Carefully collect the supernatant from the microcentrifuge tube and add it to the 1 mL syringe with attached filter.
1. Gently, filter the supernatant into the second labeled microcentrifuge tube.
1. This is your high titer phage lysate, which can be stored at 4 C.
1. Proceed with the Spot titer to estimate the phage titer

---

## Spot titer

### Before you start:
{: .no_toc }

- You need a growing culture of your bacterial host
- You need to have pre-melted molten top agar in your [bead bath]
- Add ions to the top agar to facilitate phage adsorption. (10 mM CaCL2 or MgSO4)

### Materials for spot titer
{: .no_toc }

- 1 LB agar plate per phage lysate
- 1 test tube per phage lysate
- LB top agar
- Serological pipettes
- Microcentrifuge tubes
- SM buffer

### Procedure for spot titer
{: .no_toc }

#### Day 1
{: .no_toc }

1. Label each LB plate with your initials, phage name, the date, and the notation “spot titer.”
1. Using a marker, divide the plate into nine sections and label each section 10-1 through 10-8 and use the last section as a buffer only control.
1. Add 250 uL of your appropriate host to the test tube. Pipette in 5 mL molten top agar, and immediately suck the mixture back up and plate onto the labeled LB agar plate.
1. Quickly tilt the plate in multiple directions until the top agar mixture evenly coats the agar plate.
1. Allow the plates to sit undisturbed for 10 minutes to solidify.
1. Serially dilute your HT phage lysates in SM buffer in labeled microcentrifuge tubes (10-1 to 10-8).
1. Spot 5 uL of each dilutions on the prepared bacterial lawn in the appropriately labeled section. Use 5 uL of SM buffer as a no phage control.
1. Allow the liquid to absorb into the agar for 30 minutes.
1. Incubate plates without inversion overnight.

#### Day 2
{: .no_toc }

1. Retrieve the plates from the incubator.
1. Image and record your observations in your lab notebook.
1. Calculate the appropriate titer.
1. If your titer is around or greater than 108 pfu/mL, then proceed with DNA extraction
1. If your titer is too low, then repeat the protocol with a different concentration of phages to create a webbed plate.
1. Continued low titer can also be overcome by concentrating large volumes of lysate in a concentrator column.
