---
layout: default
title: 7. Preparing high titer lysates for DNA extraction
parent: Lab Protocols
nav_order: 7
published: true
---

# Preparing high titer lysates for DNA extraction
{: .no_toc }

<details open markdown="block">
  <summary>
    Table of contents
  </summary>
  {: .text-delta }
1. TOC
{:toc}
</details>

## Objective
{: .no_toc }
To acquire a high concentration of viral particles to have enough material for whole genome sequencing of the phages.

## Rationale
{: .no_toc }
Each phage is unique and encodes its own set of genes that is variable between phage families and phages that infect different bacterial hosts. In addition to plaque assays, which tells us information about the phenotypes of these phages (such as plaque size and clear-ness), we also rely on whole genome sequencing to inform us about the genotypes of our phages, such as which genes these phages encode and what makes one phage different from another on a molecular level. In order to be able to sequence the genomes of the phages, we first need to ensure that we have enough phage particles by creating a high "titer" phage lysate. In phage terminology, the "titer" is the number of phages in a sample. High titer (HT) phage stocks can be collected from agar plates that contain large numbers of phage plaques. These plates are called confluent lysis plates, or “webbed” plates, because they have densely packed confluent plaques with only a “web” of bacteria, left between them. Depending on the phage, each plaque on a plate may contain between 10<sup>3</sup> and 10<sup>7</sup> phage particles. These phage are collected to generate a concentrated phage solution called a lysate. Lysates are collected by “flooding” a webbed plate. Flooding is accomplished by adding SM buffer to cover the surface of the plate and allowing the phage to diffuse into the buffer. The buffer is then harvested and filtered to remove agar particles and bacteria. We can estimate the titer of the lysate by dilution spot titer and this will tell us if there are enough phages to perform whole genome sequencing.

---

## Making _webbed_ confluent lysis plates

### Before you start:
{: .no_toc }
- You need a growing culture of your bacterial host
- You need to have pre-melted molten top agar in your bead bath
- You need to add ions to the top agar to facilitate phage adsorption. (Add 250 µl CaCL<sub>2</sub> to 25 mL of agar to get a final concentration of 10mM)
- You need to have an estimate of the titer of your plaque-pure phage stock

### Materials making “webbed” confluent lysis plates
{: .no_toc }
- LB agar plates (as many plates as samples to be plaqued + 1 no phage control)
- Culture tubes (as many tubes as samples to be plaqued + 1 for no phage control)
- LB top agar
- Serological pipettes
- Microcentrifuge tubes
- SM buffer
- 1 mL syringes
- 0.22 µM filters for syringes

--

### Procedure for making “webbed” confluent lysis plates
{: .no_toc }

#### Day 1
{: .no_toc }

1. Fill five microcentrifuge tubes with 90µl of SM buffer, and make five 10-fold serial dilutions of your plaque stock.
1. Label 5 LB agar plates with today's date, your name and your phage ID. Label the plates with 5 dilutions, along with your name, phage name, and the date. Label an extra plate as a no-phage control plate.
1. Set out as many culture tubes as plates. Label the culture tubes with the phage name and the phage dilution, including a no phage control tube.
1. Add the appropriate bacterial host to each labeled tube.
- {: .label .label-red }Team Coryne:
- {: .fs-3 } **250** µL of _C. glu_
- {: .label .label-purple } Team plasmid-dependent: 
- {: .fs-3 } **100** µL of _E.coli_ - pRP4
- {: .fs-3 } **100** µL of _P. putida_ - pRP4
1. Add 10 uL of the phage diltiuons to each tube, as well as 10 µL of SM buffer to the no-phage control tube. Gently vortex the tubes and allow to sit on the benchtop undisturbed for 10 minutes to allow phage adsorption.
1. Using a sterile serological pipette, gently add 4 mL molten top agar (supplemented with CaCl2) to the tubes, and immediately suck up the top agar/phage/bacteria mixture and transfer it to the appropriately labeled plates.
1. Quickly tilt the plate in multiple directions until the top agar mixture evenly coats the agar plate. 
1. Repeat this process for all of your samples, including the no phage control.
1. Let the plates sit undisturbed for 5 minutes on the benchtop to allow the top agar to fully solidify.
1. Without inverting, incubate the plates overnight in the 30 C incubator.

#### Day 2
{: .no_toc }

1. Retrieve the plates from the incubator and identify which plate(s) show the best evidence of confluent or "webbed" lysis. Write down the conditions in your lab notebook (dilution and volume of plaque stock you added)
1. Flood the lysis plates with 4 mL SM buffer, and allow the phages to diffuse into the liquid for at least 1 hour.
> You can leave the plates diffusing overnight at 4 C
3. Label two microcentrifuge tubes with the date, your inititals; "HT" and "HT-filt".
4. Retrieve the overlain plates carefully.
5. Remove the lid from the plate and place it on the bench. Tilt the plate slightly by placing one edge of the plate on the lid, allowing the lysate to pool to one side.
6. Carefully pipette 1 mL of lysate using the p1000 into the "HT" microcentrifuge tube. Try not to disturb the top agar. Carefully relid the plate and dispose of in biobin.
7. Centrifuge the tube to remove and bacteria for 3 minutes at 5,000xg.
8. Remove the plunger from a 1 mL syringe and attach a filter to the tip.
9. Carefully collect 800 µL of the supernatant from the microcentrifuge tube and add it to the 1 mL syringe with attached filter.
10. Gently, filter the supernatant into the second labeled microcentrifuge tube labelled "HT-filt".
11. This is your high titer phage lysate, which can be stored at 4 C.
12. Use the spot titer protocol (in the Phage Purification protocol- Serial dilution of phages for plaque assays) to estimate the titer of your high titer phage lysate. It should be between 10<sup>6</sup> and 10<sup>8</sup>.
