---
layout: default
title: 7. Preparing high titer lysates for DNA extraction
parent: Lab Protocols
nav_order: 7
published: true
---

# Preparing high titer lysates for DNA extraction
{: .no_toc }

<details open markdown="block">
  <summary>
    Table of contents
  </summary>
  {: .text-delta }
1. TOC
{:toc}
</details>

## Objective
{: .no_toc }
To acquire a high concentration of viral particles to have enough material for whole genome sequencing of the phages.

## Rationale
{: .no_toc }
Each phage is unique and encodes its own set of genes that is variable between phage phamilies and phages that infect different bacterial hosts. In addition to plaque assays, which tells us information about the phenotypes of these phages (such as plaque size and clear-ness), we also rely on whole genome sequencing to inform us about the genotypes of our phages, such as which genes these phages encode and what makes one phage different from another on a nucleotide level. In order to be able to sequence the genomes of the phages, we first need to ensure that we have enough phage particles by creating a high titer phage lysate. These high titer (HT) lysates can be collected by flooding plates with as many plaques as possible. These plates are called confluent lysis plates, or “webbed” plates, because they have densely packed confluent plaques with only wisps, or a “web” of bacteria, left between them. Depending on the phage, each plaque on a plate may contain between 10<sup>3</sup> and 10<sup>7</sup> phage particles. These phage are collected to generate a concentrated phage solution called a lysate. Lysates are collected by “flooding” a webbed plate. Flooding is accomplished by adding SM buffer to cover the surface of the plate and allowing the phage to diffuse into the buffer. The buffer is then harvested and filtered to remove agar particles and bacteria. We then will do a quick spot titer to estimate the PFU of the HT stock to determine if there are enough phages to perform whole genome sequencing.

---

## Making _webbed_ confluent lysis plates

### Before you start:
{: .no_toc }
- You need a growing culture of your bacterial host
- You need to have pre-melted molten top agar in your bead bath
- Add ions to the top agar to facilitate phage adsorption. (10 mM CaCL<sub>2</sub>)
- You need to have an estimate of the titer of your plaque-pure phage stock

### Materials making “webbed” confluent lysis plates
{: .no_toc }
- X number of LB agar plates (as many plates as samples to be plaqued + 1 no phage control)
- X number of test tubes (as many tubes as samples to be plaqued + 1 for no phage control)
- LB top agar
- Serological pipettes
- Microcentrifuge tubes
- SM buffer
- 1 mL syringes
- 0.22 uM filters for syringes

--

### Procedure for making “webbed” confluent lysis plates
{: .no_toc }

#### Day 1
{: .no_toc }

1. Calculate the amount of phage needed to create a “webbed” confluent lysis plate
1. Using your notes from the spot assay, determine the dilution of your single plaques needed to yeild a spot where you can no longer see individual plaques. Make this dilution again, along with two further 10-fold dilutions. (reference the spot assay protocol for how to make the dilutions)
1. Label 4 LB agar plates with today's date, your name and your phage ID. Label 3 plates with the 3 dilutions and the final plate as no phage control plate.
1. Set out as many test tubes as plates. Label the culture tubes with the phage name and the phage dilution, including a no phage control tube.
1. Add the appropriate bacterial host to each labeled tube.
- {: .label .label-red }Team Coryne:
- {: .fs-3 } **250** µL of _C. glu_
- {: .label .label-purple } Team plasmid-dependent: 
- {: .fs-3 } **100** µL of _E.coli_ - pK2
- {: .fs-3 } **100** µL of _P. putida_ - pK2
1. Add 10 uL of the phage diltiuons to each tube, as well as 10 uL of SM buffer with no phage as a control. Gently vortex the tubes and allow to sit on the benchtop undisturbed for 10 minutes to allow phage adsorption
1. Using a sterile serological pipette, gently add 3 mL molten agar supplemented with CaCl2 to a single inoculated host, and immediately suck up the top agar/phage/bacteria mixture and transfer it to the appropriately labeled plates. Be sure to avoid air bubbles, as they can look like plaques on plates.
1. Quickly tilt the plate in multiple directions until the top agar mixture evenly coats the agar plate. 
1. Repeat this process for all of your samples, including the no phage control.
1. Let the plates sit undisturbed for 20 minutes on the benchtop to allow the top agar to fully solidify.
1. Without inverting, incubate the plates overnight in the appropriate incubator.

#### Day 2
{: .no_toc }

1. Retrieve the plates from the incubator and identify which plate(s) show the best evidence of confluent lysis. Write down the conditions in your lab notebook (dilution/volume)
2. Flood the lysis plates with 4 mL SM buffer, and carefully store overnight at 4 C.

#### Day 3
{: .no_toc }

1. Label two microcentrifuge tubes with the date, your inititals; "HT" and "HT-filt".
1. Retrieve the overlain plates gently.
1. Remove the lid from the plate and place it on the bench. Tilt the plate slightly by placing one edge of the plate on the lid, allowing the lysate to pool to one side.
1. Carefully pipette 1 mL of lysate using the p1000 into the "HT" microcentrifuge tube. Try not to disturb the top agar. Carefully relid the plate and dispose of in biobin.
4. Centrifuge the tube to remove and bacteria for 3 minutes at 5,000xg.
5. Remove the plunger from a 1 mL syringe and attach a filter to the tip.
6. Carefully collect 800 uL of the supernatant from the microcentrifuge tube and add it to the 1 mL syringe with attached filter.
7. Gently, filter the supernatant into the second labeled microcentrifuge tube labelled "HT-filt".
8. This is your high titer phage lysate, which can be stored at 4 C.
9. Use the spot titer protocol to estimate the titer of your high titer phage lysate. It should be between 10<sup>6</sup> and 10<sup>8</sup>.
